examples/data-sources/scope/data.tf

```diff
@@ -0,0 +1,9 @@
+data "boundary_scope" "org" {
+  name                     = "SecOps"
+  scope_id                 = "global"
+}
+
+data "boundary_scope" "project" {
+	name = "2111"
+	scope_id = data.boundary_scope.id
```

Comment:
Since this is an example, do you mind making the indentation consistent here? Maybe just run `terraform fmt` on this.